src/stravalib/client.py

```diff
@@ -656,11 +730,25 @@ def create_activity(
         ----------
         name : str
             The name of the activity.
-        activity_type : str
+        activity_type : str, default=None
             The activity type (case-insensitive).
+            Deprecated. Prefer to use sport_type. In a request where both type
+            and sport_type are present, this field will be ignored.
+            See https://developers.strava.com/docs/reference/#api-models-UpdatableActivity.
             Possible values: ride, run, swim, workout, hike, walk, nordicski,
```

Comment:
I went ahead and refered to 

For possible values see: :class:`stravalib.model.Activity.TYPES` for these as well.